geopyspark/geotrellis/geotiff_rdd.py

```diff
@@ -89,35 +90,26 @@ def geotiff_rdd(geopysc,
         (bucket, prefix) = key_and_bucket_uri.split("/", 1)
 
         if not options:
-            result = geopysc._s3_geotiff_rdd.getRDD(key,
-                                                    bucket,
-                                                    prefix,
-                                                    geopysc.sc)
+            srdd = geopysc._s3_geotiff_rdd.getRDD(key,
```

Comment:
We should be able to bring these `_s3_geotiff_rdd` and other imports to this module, since this is the only place they are used.